sdcm/cluster_aws.py

```diff
@@ -394,7 +394,12 @@ def restart(self):
         if self._instance.spot_instance_request_id:
             logger.debug("target node is spot instance, impossible to stop this instance, skipping the restart")
             return
+
+        event_filters = ()
         if any(ss in self._instance.instance_type for ss in ['i3', 'i2']):
+            # since there's no disk yet in those type, lots of the errors here are acceptable, and we'll ignore them
+            event_filters = DbEventsFilter(type="DATABASE_ERROR"), DbEventsFilter(type="SCHEMA_FAILURE"), DbEventsFilter(type="NO_SPACE_ERROR")
```

Comment:
Error I touched and need to be ignored:
```
https://github.com/scylladb/scylla-enterprise/issues/905

sstable - Could not create SSTable component /var/lib/scylla/data/system/peers-37f71aca7dc2383ba70672528af04d4f/system-peers-ka-42-Data.db. Found exception: std::experimental::filesystem::v1::filesystem_error (error system:2, No such file or directory)
seastar - Exceptional future ignored: std::experimental::filesystem::v1::filesystem_error (error system:2, No such file or directory)
compaction_manager - compaction failed: std::experimental::filesystem::v1::filesystem_error (error system:2, No such file or directory)
sstable - Unable to delete /var/lib/scylla/data/system/peers-37f71aca7dc2383ba70672528af04d4f/system-peers-ka-42-TOC.txt because it doesn't exist.
```

Reply:
it would catch the first error,
but you did remind me we aren't catching the filesystem_error as events

Reply:
Fixed that